cinch/roles/jenkins_master/tasks/post_configure.yml

```diff
@@ -29,6 +29,10 @@
     # jenkins_security_enabled is false
     - script: "{{ lookup('template', role_path + '/templates/basic_security.groovy') }}"
       when: "{{ not jenkins_security_enabled }}"
+    # This option will automatically configure Jenkins to avoid security warning
+    # jenkins_advised_security is default as false
+    - script: "{{ lookup('template', role_path + '/templates/advised_security.groovy') }}"
+      when: "{{ jenkins_advised_security }}"
```

Comment:
What happens when this is set, but `jenkins_security_enabled` is false? Is there any problem with that permutation of this, or are these security settings independent of the ones set by `jenkins_security_enabled`?

Reply:
That's a good point. I need to test and think about this. Also, this script will disable the CLI -remoting, do you think this will conflict our work because we may need CLI -remoting on.

Reply:
The -remoting flag work was initially added by @droble (who GitHub is not allowing me to tag) when we moved to the 2.x series. It was needed for the jenkins-cli.jar file that was being used in a few places. I would defer to his knowledge on this matter to know if it is still needed and in use.

Reply:
To my knowledge *-remoting* is still needed for Jenkins slave connectivity.

Reply:
Also, I want to clarify that, this script is a personal idea to give an option for user to fix the security warnings automatically. But if conflicts with the project, or not the features you guys want, I think no need to add this.

Reply:
With the default setting as `false`, I see no reason not to add this to the project.  When Jenkins slave consumers are ready to migrate to whatever option doesn't need *-remoting*, the variable can be set to `true`.